bin/metaBackend.py

```diff
@@ -21,47 +21,506 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 
 """
-This is a program for ingesting information into LSST Metadata Server.
+Metadata Server client program. It is currently used to ingest information into the
```

Comment:
Script name is confusing, can we rename it to something like `metaClient` or `metaAdmin`?